### PR TITLE
docs: various documentation improvements

### PR DESCRIPTION
**Pull request by ra0x3**
### Description
- Took a look over the book/docs and just updated a few things that I found

### Testing steps
- N/A

### Changelog 
- docs: various documentation improvements

### CODE REVIEW

File: docs/QUICKSTART.md [hunk 1]
```diff
@@ -235,6 +236,10 @@ http://127.0.0.1:29987/api/graph/my_project/hello_indexer
 ]
 ```
 
+### 3.1 Using the playgrond
+
+<< MENTION YOU CAN USE THE PLAYGROUND TO WRITE QUERIES >>
+
```
Comment (deekerno) [inline docs/QUICKSTART.md:242]: This should be removed or adjusted.

File: docs/README.md [hunk 2]
```diff
@@ -8,18 +8,21 @@
     </picture>
 
 </p>
-<p align="center">
+<p align="left">
```
Comment (deekerno) [inline docs/README.md:11]: The image shields are now left-aligned while the header and architecture images are center-aligned.

**Review by deekerno — CHANGES_REQUESTED** [review]

**Review by deekerno — APPROVED** [review]
:ok-dude:

File: docs/QUICKSTART.md [hunk 3]
```diff
@@ -3,16 +3,16 @@
 In this tutorial you will:
 
 1. Bootstrap your development environment.
-2. Create, build, and deploy an index to an indexer service hooked up to Fuel's `beta-3` testnet.
-3. Query the indexer service for indexed data using GraphQL.
+2. Create, build, and deploy an indexer to an indexer service hooked up to Fuel's `beta-3` testnet.
```
Comment (rvmelkonian) [inline docs/QUICKSTART.md:6]: -er it. for ev-er

File: docs/QUICKSTART.md [hunk 4]
```diff
@@ -24,21 +24,21 @@ curl \
 
 ### 1.2 WebAssembly (WASM) Setup
 
-Indexers are typically compiled to WASM and thus you'll need to have the proper WASM compilation target available on your system. You can install it through `rustup`:
+Indexers are typically compiled to WASM so you'll need to have the proper WASM compilation target available on your system. You can install this target using `rustup`:
```
Comment (rvmelkonian) [inline docs/QUICKSTART.md:27]: I think this more friendly language is a good call.

File: docs/src/reference-guide/authentication/index.md [hunk 5]
```diff
@@ -20,11 +20,10 @@ Please enter your password:
...
 
-
 ```text
 ✅ Successfully authenticated at https://indexer.fuel.network:29987/api/auth/signature.
 
 Token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiODNlNjhiOTFmNDhjYWM4M....
 ```
 
-Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way.
+Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way. Further, users can just pass this JWT token value to the `--auth` flag, if using `forc index` commands that support authentication (e.g., `forc index deploy --auth $MY_JWT_TOKEN`).
```
Comment (rvmelkonian) [inline docs/src/reference-guide/authentication/index.md:29]: Very Small nit. But I think the text might be more readable to break this into to small paragraphs.  We can also remove the connecting word if we do this 'Further,'.  Neither is right or wrong, the thought just occurred to me.

"Use this token in your `Authorization` headers when making requests for operations such as uploading indexers, stopping indexers, and other operations that mutate state in this way.

Users can just pass this JWT token value to the `--auth` flag, if using `forc index` commands that support authentication (e.g., `forc index deploy --auth $MY_JWT_TOKEN`)."

**Review by rvmelkonian — DISMISSED** [review]
One small comment. But it's not a must change. Good improvements here.

File: docs/QUICKSTART.md (hunk 3 above)
Comment (ra0x3) [reply docs/QUICKSTART.md:6]: I'm not sure what this means

Comment (rvmelkonian) [reply docs/QUICKSTART.md:6]: We are forever adding 'er' on to the word index. Too much screen time today for me!

**Review by rvmelkonian — APPROVED** [review]

**Pull request merged**